`deno_os`: `sysinfoapi` feature flag on `winapi` dependency not enabled, can't build the crate

Version: Deno 0.203.0

Thank you for making Deno, it's super awesome :)

I noticed something about `deno_os`; though it uses `winapi::um::sysinfoapi`, the `sysinfoapi` feature flag on the `winapi` crate doesn't seem to be enabled? Fixing this should be as simple as adding the feature flag into `ext/os/Cargo.toml`.

Given how this affects building the crate I was surprised to see that no one has reported this yet, but thought I'd report it anyways.

## Reproducing

1. Make a new Rust binary project
2. Use the following as the `Cargo.toml` file

```toml
[package]
name = "simple-test"
version = "0.1.0"
edition = "2024"

[dependencies]
deno_os = "0.12"
```

## Expected

Should build ok.

## Actual result

```
    Checking deno_permissions v0.54.0
    Checking deno_telemetry v0.17.0
    Checking deno_os v0.12.0
error[E0432]: unresolved import `winapi::um::sysinfoapi`
   --> C:\Users\---\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\deno_os-0.12.0\sys_info.rs:315:9       
    |
315 |     use winapi::um::sysinfoapi;
    |         ^^^^^^^^^^^^^^^^^^^^^^ no `sysinfoapi` in `um`
    |
note: found an item that was configured out
   --> C:\Users\---\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\winapi-0.3.9\src\um\mod.rs:236:40      
    |
236 | #[cfg(feature = "sysinfoapi")] pub mod sysinfoapi;
    |                                        ^^^^^^^^^^
note: the item is gated behind the `sysinfoapi` feature
   --> C:\Users\---\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\winapi-0.3.9\src\um\mod.rs:236:7       
    |
236 | #[cfg(feature = "sysinfoapi")] pub mod sysinfoapi;
    |       ^^^^^^^^^^^^^^^^^^^^^^

error[E0433]: failed to resolve: could not find `sysinfoapi` in `um`
   --> C:\Users\---\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\deno_os-0.12.0\sys_info.rs:425:26      
    |
425 |     uptime = winapi::um::sysinfoapi::GetTickCount64() / 1000;
    |                          ^^^^^^^^^^ could not find `sysinfoapi` in `um`
    |
note: found an item that was configured out
   --> C:\Users\---\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\winapi-0.3.9\src\um\mod.rs:236:40      
    |
236 | #[cfg(feature = "sysinfoapi")] pub mod sysinfoapi;
    |                                        ^^^^^^^^^^
note: the item is gated behind the `sysinfoapi` feature
   --> C:\Users\---\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\winapi-0.3.9\src\um\mod.rs:236:7       
    |
236 | #[cfg(feature = "sysinfoapi")] pub mod sysinfoapi;
    |       ^^^^^^^^^^^^^^^^^^^^^^

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `deno_os` (lib) due to 2 previous errors
```

As expected if the dependency is added manually in the test project like so to patch in the feature:
```
winapi = { version = "0.3.9", features = ["sysinfoapi"] }
```
This error goes away.

--- a/ext/os/Cargo.toml
+++ b/ext/os/Cargo.toml
@@ -27,7 +27,7 @@ thiserror.workspace = true
 tokio.workspace = true
 
 [target.'cfg(windows)'.dependencies]
-winapi = { workspace = true, features = ["commapi", "knownfolders", "mswsock", "objbase", "psapi", "shlobj", "tlhelp32", "winbase", "winerror", "winuser", "winsock2"] }
+winapi = { workspace = true, features = ["commapi", "knownfolders", "mswsock", "objbase", "psapi", "shlobj", "sysinfoapi", "tlhelp32", "winbase", "winerror", "winuser", "winsock2"] }
 ntapi = "0.4.0"
 
 [target.'cfg(unix)'.dependencies]

